src/app/services/backups.js

```diff
@@ -163,13 +163,30 @@ module.exports = (Model, App) => {
 
     const folders = await Model.folder.findAll({ where: { bucket: backupsBucket } });
 
-    return Promise.all(
-      folders.map(async (folder) => ({
-        ...folder.get({ plain: true }),
-        hasBackups: !(await isDeviceAsFolderEmpty(folder)),
-        lastBackupAt: folder.updatedAt,
-      })),
+    const newFolders = await Promise.all(
+      folders.map(async (folder) => {
+        const decryptedWithBucket = App.services.Crypt.decryptName(folder.name, folder.bucket);
+
+        const shouldUpdateName = !decryptedWithBucket && folder.bucket;
+
+        const backupName = shouldUpdateName
+          ? App.services.Crypt.encryptName(App.services.Crypt.decryptNameWithNullFolderId(folder.name), folder.bucket)
+          : folder.name;
+
+        if (shouldUpdateName) {
+          await folder.update({ name: backupName }, { silent: true });
```

Comment:
I noticed the frontend uses updated_at to show the last modification on the backups section. 

Added silent just to prevent `updated_at` being updated and mess with user's timestamps (however, `updated_at` should be updated by the backend in these cases too)